docs/http-clients.md

```diff
@@ -0,0 +1,17 @@
+# HTTP Clients
+
+To date, we have four different HTTP API clients:
+
+- [kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client) - Official JS client for talking to Kubo over HTTP
+- [ipfs-http-client](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-client) - http client for js-ipfs -- Use when talking to [`js-ipfs`](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs)
+- [go-ipfs-api](https://github.com/ipfs/go-ipfs-api) - The go interface to ipfs's HTTP API
+- [go-ipfs-http-client](https://github.com/ipfs/go-ipfs-http-client) - IPFS CoreAPI implementation using HTTP API
```

Comment:
Let's get a link to where we're cleaning this up.

Reply:
Is there an existing tracking issue for the go-rpc-client cleanup or does one need to be created? Should I use https://github.com/ipfs/kubo/issues/9142 ?

Reply:
https://github.com/ipfs/kubo/issues/9142 has https://github.com/ipfs/kubo/issues/9124.  I think you can use #9124.
(funny how those numbers are very similar!)